src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,113 @@
+| `SELECT ... FOR UPDATE / FOR SHARE` | ✓                                                   | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT / MERGE ... THEN INSERT`    | —                                                   | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`              | ✓ ‡                                                 | ✓                                                | —                                                 | —                                                     | —                                             |
+| `UPDATE / MERGE ... THEN UPDATE`    | ✓ ‡                                                 | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `DELETE`                            | ✓ ‡                                                 | —                                                | —                                                 | —                                                     | ✓                                             |
+| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+
+‡ The `SELECT` check is only evaluated when the statement actually needs to read from the relation, e.g., in a `WHERE`, `SET`, or `RETURNING` clause that references table columns.
+
+CockroachDB's behavior differs from that described in the table above in the following ways:
+
+- `SELECT` evaluation: CockroachDB always evaluates `SELECT` (`USING`) policies for `INSERT` and `UPDATE`, even when the statement doesn't reference table columns. This differs from PostgreSQL; this difference may be resolved in an upcoming release.
+- `ON CONFLICT ... DO NOTHING`: CockroachDB still runs constraint and row-level policy checks on the `VALUES` clause even when the candidate row is discarded because of a conflict, so `INSERT ... ON CONFLICT DO NOTHING` may reject rows that PostgreSQL would silently ignore. This is a known limitation described in [cockroachdb/cockroach#35370](https://github.com/cockroachdb/cockroach/issues/35370).
```

Comment:
Can you link directly to the KL on the Known Limitations doc, or to a Known Limitations heading on this page (if it exists)?

Reply:
i have created a new heading on the KL page, a new KL section on this page, and an include to share the info into the various places